packages/next-templates/src/app/page.tsx

```diff
@@ -32,21 +36,36 @@ export default function Home() {
       <ExampleNavigation />
       <PageContent className="voorbeeld-page-content--flex">
         <section className="voorbeeld-section">
-          <TopTaskNavigation className="grid">
-            <TopTaskLink icon={<UtrechtIconPaspoort />}>Paspoort of ID-kaart aanvragen</TopTaskLink>
-            <TopTaskLink href="/meldingen/" icon={<UtrechtIconMeldingKlacht />}>
-              Meldingen openbare ruimte
-            </TopTaskLink>
-            <TopTaskLink icon={<UtrechtIconVerhuizen />}>Verhuizing doorgeven</TopTaskLink>
-            <TopTaskLink icon={<UtrechtIconWerken />}>Werken bij de gemeente</TopTaskLink>
-            <TopTaskLink icon={<UtrechtIconNummerbord />}>Parkeren: kentekenwijziging doorgeven</TopTaskLink>
-            <TopTaskLink icon={<UtrechtIconAfvalScheiden />}>Afval</TopTaskLink>
-          </TopTaskNavigation>
+          <div className="voorbeeld-section-content voorbeeld-toptask-layout">
+            <TopTaskNavigation className="grid">
+              <TopTaskLink icon={<UtrechtIconPaspoort />}>Paspoort of ID-kaart aanvragen</TopTaskLink>
+              <TopTaskLink href="/meldingen/" icon={<UtrechtIconMeldingKlacht />}>
+                Meldingen openbare ruimte
+              </TopTaskLink>
+              <TopTaskLink icon={<UtrechtIconVerhuizen />}>Verhuizing doorgeven</TopTaskLink>
+              <TopTaskLink icon={<UtrechtIconWerken />}>Werken bij de gemeente</TopTaskLink>
+              <TopTaskLink icon={<UtrechtIconNummerbord />}>Parkeren: kentekenwijziging doorgeven</TopTaskLink>
+              <TopTaskLink icon={<UtrechtIconAfvalScheiden />}>Afval</TopTaskLink>
+            </TopTaskNavigation>
+            <TopTaskHeader>
+              <IconClock />
+              <Heading3>Openingstijden</Heading3>
+              <Heading4>Openingstijden Gemeentehuis</Heading4>
+              <Paragraph className="voorbeeld-paragraph-opening-times">
+                Vandaag:
+                <br />
+                08.30 - 17.00
+                <br />
+                18.00 - 20.00
+              </Paragraph>
+              <Button appearance="secondary-action-button">Contact</Button>
+            </TopTaskHeader>
+          </div>
         </section>
         <section className="voorbeeld-section voorbeeld-section--spacing">
           <div className="voorbeeld-section-content">
             <div className="voorbeeld-section-container-wrapper">
-              <Heading2 className="voorbeeld-heading-2">Zelf regelen</Heading2>
+              <Heading2>Zelf regelen</Heading2>
               <Button className="voorbeeld-button-more" appearance="primary-action-button">
                 Meer bekijken
               </Button>
```

Comment:
Zullen we een `ButtonGroup` om de `Button` heen doen?
Zou het echt een `Button` zijn, als het geen formulier is?